### DISCUSSION TITLE
GitHub Support has not responded to my ticket for 69 days.

### DISCUSSION THREAD

**Discussion by sandulat** [community/community#83929] [Other Feature Feedback, Questions, & Ideas]
I created an NPM package name dispute ticket on October 25, 2023, to request the name of a completely empty package that was published 8 years ago. I've heard that it usually takes 1-2 months to resolve these kinds of requests, but we are now approaching the third month of waiting. Where can I seek assistance?

**Comment by sandulat** [comment]
72 days now. Help 😭.

**Reply by manarea** [reply]
Did they answer you already?
